.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Expected behavior vs actual behavior
+<br><br>
+
+#### Steps to reproduce
+*(e.g., detailed walkthrough, runnable script, example application)*
+<br><br><br>
+
+#### Environment
+
+AMS Version *(commit ref if not on tag)*: 
```

Comment:
> I figured we could revisit it if it got to that point.

totally. just thinking 'aloud'.  Bundler has an interesting built-in error report https://github.com/bundler/bundler/blob/f9a3a5148e7902ae4e30e837060806067e9025f5/lib/bundler/friendly_errors.rb#L53-L95